workflows/utilities/wf_read_QC_trim_ont.wdl

```diff
@@ -46,13 +46,25 @@ workflow read_QC_trim_ont {
         samplename = samplename,
         read1 = read1,
         target_org = target_org
+    }
+    call kraken2.kraken2_parse_classified as kraken2_recalculate_abundances_raw {
+    input:
+      samplename = samplename,
+      kraken2_report = kraken2_raw.kraken_report,
+      kraken2_classified_report = kraken2_raw.kraken2_classified_report
     }  
     call kraken2.kraken2_theiacov as kraken2_dehosted {
       input:
         samplename = samplename,
         read1 = ncbi_scrub_se.read1_dehosted,
         target_org = target_org
     }
+    call kraken2.kraken2_parse_classified as kraken2_recalculate_abundances_dehosted {
+    input:
+      samplename = samplename,
+      kraken2_report = kraken2_dehosted.kraken_report,
+      kraken2_classified_report = kraken2_dehosted.kraken2_classified_report
```

Comment:
you're not passing in the target organism to this task? also please rename target_org to target_organism to match standardization

Reply:
Thanks for the great catch!